tests/forms_tests/tests/test_formsets.py

```diff
@@ -877,6 +884,67 @@ def test_more_initial_than_max_num(self):
 <td><input id="id_form-2-name" name="form-2-name" type="text" value="Jack and Coke"></td></tr>"""
         )
 
+    def test_default_absolute_max(self):
+        """
+        The default absolute max is 2*formsets.DEFAULT_MAX_NUM
+        """
+        data = {
+            'form-TOTAL_FORMS': '2505',
+            'form-INITIAL_FORMS': '0',
+            'form-MAX_NUM_FORMS': '0',
+        }
+
+        # The default absolute_max is max_num + DEFAULT_MAX_NUM (which is
+        # 2*DEFAULT_MAX_NUM if max_num isn't specified)
+        formset = FavoriteDrinksFormSet(data=data)
+        self.assertFalse(formset.is_valid())
+        self.validate_formset_number_of_forms(formset, 2 * formsets.DEFAULT_MAX_NUM)
```

Comment:
This test works without a patch, please move it to a separate commit/PR, we should also add assertion for `absolute_max`, e.g.
```python
    def test_default_absolute_max(self):
        # absolute_max defaults to 2 * DEFAULT_MAX_NUM if max_num is None.
        data = {
            'form-TOTAL_FORMS': 2001,
            'form-INITIAL_FORMS': '0',
            'form-MAX_NUM_FORMS': '0',
        }
        formset = FavoriteDrinksFormSet(data=data)
        self.assertIs(formset.is_valid(), False)
        self.assertEqual(
            formset.non_form_errors(),
            ['Please submit 1000 or fewer forms.'],
        )
        self.assertEqual(formset.absolute_max, 2000)
```